internal/api.go

```diff
@@ -173,6 +156,12 @@ func (c *APIClient) Post(url, contentType string, body io.ReadSeeker) (resp *htt
 	// transfer encoding.
 	if rb, ok := body.(*RingBuffer); ok {
 		req.ContentLength = int64(rb.Len())
+		// Fix the content-type header to avoid chunked transfer encoding
```

Comment:
Can you also fix the comment 7 lines above?

> [...] automatically sets content-type

It should be `content-length`.